who-we-are/team/max-carlson.md

```diff
@@ -0,0 +1,11 @@
+
+role: Code Steward, North America
+image: max-carlson.jpeg
+
+---
+
+Max is the Code Steward for the Foundation for Public Code North America.
```

Comment:
Is it meant to be Codebase Steward like in the umbrella organization, or is this intentionally different?